actuators/machine.go

```diff
@@ -197,41 +191,41 @@ func (m *Machine) Exists(ctx context.Context, cluster *clusterv1.Cluster, machin
 		fmt.Sprintf("label=%s=%s", constants.ClusterLabelKey, cluster.Name),
 		fmt.Sprintf("name=^%s$", machine.GetName()),
 	}
-	fmt.Printf("using labels: %v\n", labels)
+	m.Log.Info("using labels", "labels", labels)
 	nodeList, err := nodes.List(labels...)
 	if err != nil {
 		return false, err
 	}
-	fmt.Printf("found nodes: %v\n", nodeList)
+	m.Log.Info("found nodes", "nodes", nodeList)
 	return len(nodeList) >= 1, nil
 }
 
 // patches the object and saves the status.
 func (m *Machine) save(old, new *clusterv1.Machine, noderef *apicorev1.ObjectReference) error {
-	fmt.Println("updating machine")
+	m.Log.Info("updating machine")
 	p, err := patch.NewJSONPatch(old, new)
 	if err != nil {
-		fmt.Printf("%+v\n", err)
+		m.Log.Error(err, "Error updating machine")
 		return err
 	}
-	fmt.Println("Patches for machine", p)
+	m.Log.Info("Patches for machine", "patches", p)
 	if len(p) != 0 {
 		pb, err := json.MarshalIndent(p, "", "  ")
 		if err != nil {
-			fmt.Printf("%+v\n", err)
+			m.Log.Error(err, "Error marshalling machine")
 			return err
 		}
 		new, err = m.ClusterAPI.Machines(old.Namespace).Patch(new.Name, types.JSONPatchType, pb)
```

Comment:
I try to avoid using go keywords as variable names. Instead of calling the updated machine `new`, how about `updatedMachine`, `patchedMachine`, or `updated`?

Reply:
yeah, agreed, but out of scope for this PR, it will get cleaned up eventually